Amazon.IonObjectMapper.Demo/FormatTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * Copyright (c) Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+namespace Amazon.IonObjectMapper.Demo
+{
+    using System;
+    using System.IO;
+    using System.Text;
+    using Amazon.IonDotnet;
+    using Amazon.IonDotnet.Builders;
+    using Amazon.IonObjectMapper.Test;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using static Amazon.IonObjectMapper.IonSerializationFormat;
+
+    [TestClass]
+    public class FormatTest
+    {
+        [TestMethod]
+        public void Scratch()
+        {
+            IonSerializer ionSerializer;
+            MemoryStream stream;
+
+            ionSerializer = new IonSerializer(new IonSerializationOptions {Format = IonSerializationFormat.BINARY});
+            stream = (MemoryStream)ionSerializer.Serialize(TestObjects.a90);
+            Console.WriteLine(string.Join(", ", stream.ToArray()));
```

Comment:
We could use `Console.WriteLine(BitConverter.ToString(stream.ToArray()));` which will show hexadecimal string representation of the Ion binary dump that is easier to read https://amzn.github.io/ion-docs/docs/binary.html

Reply:
Updated